lib/kubeclient/config.rb

```diff
@@ -0,0 +1,111 @@
+require 'yaml'
+require 'base64'
+
+module Kubeclient
+  class Config
+    class Context
+      attr_reader :api_endpoint, :api_version, :ssl_options
+
+      def initialize(api_endpoint, api_version, ssl_options)
+        @api_endpoint = api_endpoint
+        @api_version = api_version
+        @ssl_options = ssl_options
+      end
+    end
```

Comment:
@jonmoter yes that would have been nice but since this part should work also with `openshift_client` we should come up with something smarter (what client to return, etc.). At the moment I went with the simple implementation where we don't prepare any of the two clients.